SQL/beestation_schema.sql

```diff
@@ -568,6 +568,14 @@ CREATE TABLE `achievement_metadata` (
 	PRIMARY KEY (`achievement_key`)
 ) ENGINE=InnoDB;
 
+DROP TABLE IF EXISTS `metacoin_item_purchases`;
+CREATE TABLE IF NOT EXISTS `metacoin_item_purchases` (
+	`ckey` varchar(32) NOT NULL,
+	`purchase_date` datetime NOT NULL DEFAULT current_timestamp() ON UPDATE current_timestamp(),
```

Comment:
what use does that date have?

Reply:
@Crossedfall

Reply:
To show the date of purchase, and to update on an additional purchase. It serves a mostly analytical function but it could be used if we ever want to add consumables that come with a cooldown.